lib/config/src/expression_functions.rs

```diff
@@ -412,10 +411,14 @@ impl Epoch {
 
     #[allow(clippy::unnecessary_wraps)]
     pub(super) fn evaluate<'a>(self) -> Result<Cow<'a, json::Value>, ExecutingExpressionError> {
-        let start = SystemTime::now();
-        let since_the_epoch = start
+        // https://github.com/rustwasm/wasm-pack/issues/724#issuecomment-776892489
+        // SystemTime is not supported by wasm-pack. So for wasm-pack builds, we'll use js_sys::Date
+        #[cfg(not(target_arch = "wasm32"))]
+        let since_the_epoch = SystemTime::now()
```

Comment:
This could be a little cleaner if you use the [cfg macro](https://doc.rust-lang.org/std/macro.cfg.html)

Reply:
Thanks @EliSnow. That makes it simpler.